crm_compassion/model/event_compassion.py

```diff
@@ -309,3 +337,33 @@ def _find_parent_analytic(self):
                 'parent_id': categ_id
             }).id
         return acc_id
+
+    def _get_calendar_vals(self):
+        """
+        Gets the calendar event values given the event
+        :return: dictionary of calendar.event values
+        """
+        self.ensure_one()
+        number_of_days = 1
+        start_date = fields.Datetime.from_string(self.start_date)
+        if self.end_date:
+            end_date = fields.Datetime.from_string(self.end_date)
+            number_of_days = (end_date - start_date).days
+        calendar_vals = {
+            'name': self.name,
+            'categ_ids': [
+                (6, 0, [self.env.ref('crm_compassion.calendar_event').id])],
+            'duration': number_of_days * 8,

```

Comment:
Not sure here to understand why you multiple the number of day by 8?

The rest is OK for me